elastic/kibana
# [Security Solution] Entities details tab in expandable flyout

## Summary

This PR adds content to the 'Entities' tab under ' Insights', in the left section of the expandable flyout. 

- User info contains an user overview and related hosts. Related hosts are hosts this user has successfully authenticated after alert time
- Host info contains a host overview and related users. Related users are users who are successfully authenticated to this host after alert time
- User and host risk scores are displayed if kibana user has platinum license

![image](https://user-images.githubusercontent.com/18648970/234703183-a3fa7809-cc1f-4b9a-8bd0-aa2a991047cb.png)

### How to test

- Enable feature flag `securityFlyoutEnabled`
- Navigation:
   - Generate some alerts data and go to Alerts page
   - Select the expand icon for an alert
   - Click `Expand alert details`
   - Go to Insights tab, Entities tab
- To see risk score, apply platinum or enterprise license, then go to dashboard -> entity analytics, and click Enable (both user and host).
- See comments below on generating test data (if needed)

### Run tests and storybook
- `node scripts/storybook security_solution` to run Storybook
- `npm run test:jest --config ./x-pack/plugins/security_solution/public/flyout` to run the unit tests
- `yarn cypress:open-as-ci` but note that the integration/e2e tests have been written but are now skipped because the feature is protected behind a feature flag, disabled by default. To check them, add `'securityFlyoutEnabled'` [here](https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_cypress/config.ts#L50)

### Checklist
